src/test/java/randoop/main/minimizer/MinimizerTests.java

```diff
@@ -0,0 +1,240 @@
+package randoop.main.minimizer;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import org.junit.Test;
+import randoop.main.Minimize;
+
+public class MinimizerTests {
+  // Obtain system separators for path and file
+  private static final String pathSeparator = System.getProperty("path.separator");
+  private static final String fileSeparator = System.getProperty("file.separator");
+
+  // Directory to test inputs
+  private static final String testDir = "test" + fileSeparator + "minimizer" + fileSeparator;
+
+  /**
+   * Test the minimizer with an input file
+   *
+   * @param inputFilePath input file to minimize: a JUnit test suite
+   * @param dependencies dependencies needed to compile and run the input file. This parameter is an
+   *     array of Strings, each representing an element of the classpath, for instance, a directory
+   *     or a jar file.
+   * @param timeoutLimit maximum number of seconds allowed for any test case within the input file
+   *     to run
+   * @throws IOException thrown if output or expected output files can't be read
+   */
+  private void testWithInput(String inputFilePath, String[] dependencies, String timeoutLimit)
+      throws IOException {
+    String outputFilePath =
+        new StringBuilder(inputFilePath)
+            .insert(inputFilePath.lastIndexOf('.'), "Minimized")
+            .toString();
+    String expectedFilePath = inputFilePath + ".expected";
+
+    // Obtain File object references
+    File inputFile = new File(inputFilePath);
+    File outputFile = new File(outputFilePath);
+
+    // Obtain the complete path to the input, output, and expected files
```

Comment:
Has this not been fixed?  This looks odd to me.  Create a File object from String and work with that.